### ISSUE TITLE
Leading zero crash

### ISSUE THREAD

**Issue by sanderschnydrig** [ionic-team/trapeze#132]
Hi Max!

I recently migrated our project from @capacitor/assets to this one ("@trapezedev/configure": "^6.0.6"), looks very promising. However I got an error message when trying to read an environment variable with the value "0", as JSON cannot parse this.

This is the workaround I'm currently applying. I was unable to open a PR here so I'll link my fork here:

https://github.com/ionic-team/trapeze/compare/main...sanderschnydrig:trapeze:main

Cheers

**Comment by sanderschnydrig** [comment]
Figured out how to create a PR 🎉 https://github.com/ionic-team/trapeze/pull/133

**Comment by mlynch** [comment]
Can you give me an example of the input that is breaking and the error message you're getting? I'm trying to understand what's going on to help review #133. I'm not able to reproduce it locally so I'm trying to understand. Thanks

**Comment by sanderschnydrig** [comment]
Sure, this is an example of my .env file:

```.env
APP_NAME = "Example"
APP_IDENTIFIER = "com.example.app"
APP_DESCRIPTION = "Example description"
APP_VERSION="2.8.16"
BUILD_NUMBER="0"
BUILD_NUMBER_ANDROID="2081600"
```

The line with the BUILD_NUMBER variable breaks this tool for me when using the following configuration (relevant parts are the BUILD_NUMBER in `vars` and the usage of it in `ios` and `android`):

```yaml
# REF: https://github.com/ionic-team/capacitor-configure/blob/main/examples/basic.yml
vars:
  APP_NAME:
  APP_VERSION:
  APP_IDENTIFIER:
  APP_DESCRIPTION:
  TEAM_ID:
  BASE_URL:
  NEWSAPP_PAGE_LINK:
  BUILD_NUMBER:
  BUILD_NUMBER_ANDROID:
  # PROVISIONING_PROFILE_SPECIFIER:

platforms:
  ios:
    targets:
      App:
        bundleId: $APP_IDENTIFIER
        version: $APP_VERSION
        buildNumber: $BUILD_NUMBER
        productName: $APP_NAME
        displayName: $APP_DESCRIPTION

        buildSettings:
          DEVELOPMENT_TEAM: $TEAM_ID
          PROVISIONING_PROFILE_SPECIFIER: 'match AppStore $APP_IDENTIFIER'

        entitlements:
          replace: true
          entries:
            - com.apple.developer.associated-domains: ['applinks:$BASE_URL', 'applinks:$NEWSAPP_PAGE_LINK']



  android:
    packageName: $APP_IDENTIFIER
    appName: $APP_DESCRIPTION
    versionName: $APP_VERSION
    versionCode: $BUILD_NUMBER_ANDROID

    res:
      - path: values
        file: strings.xml
        text: |
            <?xml version='1.0' encoding='utf-8'?>
            <resources>
                <string name="app_name">$APP_NAME</string>
                <string name="title_activity_main">$APP_NAME</string>
                <string name="package_name">$APP_IDENTIFIER</string>
                <string name="custom_url_scheme">$APP_IDENTIFIER</string>
            </resources>

      - path: values-v29
        file: strings.xml
        text: |
            <?xml version='1.0' encoding='utf-8'?>
            <resources>
                <string name="app_name">$APP_NAME</string>
                <string name="title_activity_main">$APP_NAME</string>
                <string name="package_name">$APP_IDENTIFIER</string>
                <string name="custom_url_scheme">$APP_IDENTIFIER</string>
            </resources>
```

**Comment by mlynch** [comment]
Thanks, will look into this and review your PR soon

**Comment by mlynch** [comment]
Fixed in #142
